hs/src/Reach/Connector/ALGO.hs

```diff
@@ -2048,6 +2049,7 @@ ce = \case
     let objLen = typeSizeOf obj_t
     ca obj_a
     ca val_a
+    ctobs $ argTypeOf val_a
     csplice at valStart (valStart + valLen) objLen
```

Comment:
I think for code cleanliness, the Tuple and Object cases should be merged together with the Object ones doing some light processing before to get the data that the tuple ones need. I suspect that `DLE_ObjectRef` can also easily use `cTupleRef` (or rather... that there's a function inside `cTupleRef` that it could use)

Reply:
I see what you're getting at

Reply:
cTupleSet already existed.. who knew